ltc/main_test.go

```diff
@@ -34,4 +35,99 @@ var _ = Describe("lattice-cli", func() {
+			Consistently(session.Out).ShouldNot(gbytes.Say("Unknown flag \"not-a-flag\""))
+		})
+
+		It("checks flags with prefix '-'", func() {
+			command := exec.Command(cli, "create", "not-a-flag", "-invalid-flag")
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say("\"-invalid-flag\""))
+			Consistently(session.Out).ShouldNot(gbytes.Say("\"not-a-flag\""))
+		})
+
+		It("checks flags but ignores the value after '=' ", func() {
+			command := exec.Command(cli, "create", "-i=1", "-invalid-flag=blarg")
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say("\"-invalid-flag\""))
+			Consistently(session.Out).ShouldNot(gbytes.Say("Unknown flag \"-p\""))
+		})
+
+		It("outputs all unknown flags in single sentence", func() {
+			command := exec.Command(cli, "create", "--bad-flag1", "--bad-flag2", "--bad-flag3")
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say("\"--bad-flag1\", \"--bad-flag2\", \"--bad-flag3\""))
+		})
+
+		It("only checks input flags against flags from the provided command", func() {
+			command := exec.Command(cli, "create", "--instances", "--skip-ssl-validation")
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session.Out).Should(gbytes.Say("\"--skip-ssl-validation\""))
+		})
+
+		It("accepts -h and --h flags for all commands", func() {
+			command := exec.Command(cli, "create", "-h")
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Consistently(session.Out).ShouldNot(gbytes.Say("Unknown flag \"-h\""))
+			command = exec.Command(cli, "target", "--h")
+			session, err = gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Consistently(session.Out).ShouldNot(gbytes.Say("Unknown flag \"--h\""))
+		})
+
+		Context("When a negative integer is preceeded by a valid flag", func() {
+			It("skips validation for negative integer flag values", func() {
+				command := exec.Command(cli, "create", "-i", "-10")
+				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(session.Out).ShouldNot(gbytes.Say("\"-10\""))
+			})
+		})
+
+		Context("When a negative integer is preceeded by a invalid flag", func() {
+			It("validates the negative integer as a flag", func() {
+				command := exec.Command(cli, "create", "-badflag", "-10")
+				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(session.Out).Should(gbytes.Say("\"-badflag\""))
+				Eventually(session.Out).Should(gbytes.Say("\"-10\""))
+			})
+		})
+	})
```

Comment:
From https://travis-ci.org/cloudfoundry-incubator/lattice/jobs/62686888, it looks like these new tests rely on a live targeted receptor endpoint.  We restrict integration-style tests to `ltc test` (integration_test_runner.go) and expect our unit tests to run in isolation from the cluster.

It looks like the `targetVerifier` (injected via `CliAppFactory`) is attempting to establish a connection to receptor during these test cases.